tests/acceptance/variant_view.js

```diff
@@ -96,85 +96,37 @@ function checkNoDataAvailable(sectionName, testName, element, elementID, expecte
     });
 }
 
-test.describe('Variant View - rs exclusive to EVA', function() {
-    var driver;
-    test.before(function() {
-        driver = config.initDriver(config.browser());
-        driver.get(config.baseURL()+'?variant&accessionID=rs2020677&species=mmusculus_grcm38');
-    });
-
-    test.after(function() {
-        config.shutdownDriver(driver);
-    });
-
-    var expectedResults = [{"Organism": "Mouse", "Assembly": "GCA_000001635.6", "Contig": "7",
-                            "Start": "119786918", "ID": "rs2020677", "Type": "",
-                            "Created Date": ""}];
-    runTableTest("Variant Information Section", "Variant Information Section has the correct values for attributes",
-                "table", "variant-view-summary", expectedResults, checkSection);
-
-    expectedResults = [{"ID": "ss47184058", "Submitter Handle": "", "Contig": "7", "Start": "119786918",
-                        "End": "119786918", "Submitter Handle": "", "Reference": "G", "Alternate": "A", "Created Date": ""},
-                        {"ID": "ss1565899", "Submitter Handle": "", "Contig": "7", "Start": "119786918",
-                         "End": "119786918", "Submitter Handle": "", "Reference": "G", "Alternate": "A", "Created Date": ""},
-                        {"ID": "ss372922602", "Submitter Handle": "", "Contig": "7", "Start": "119786918",
-                         "End": "119786918", "Reference": "G", "Alternate": "A", "Created Date": ""}
-                      ];
-    runTableTest("Submitted Variant Section", "Submitted Variant Section has the correct values for attributes",
-                "table", "submitted-variant-summary", expectedResults, checkSection);
-});
-
 test.describe('Variant View - ss exclusive to EVA', function() {
     var driver;
     test.before(function() {
         driver = config.initDriver(config.browser());
-        driver.get(config.baseURL()+'?variant&accessionID=ss914406059&species=mmusculus_grcm38');
+        driver.get(config.baseURL()+'?variant&accessionID=vcZ2JUAF0&species=zmays_agpv3');
```

Comment:
I'm not sure this makes sense anymore, because we are not planning to support searches by non-(ss/rs) identifiers.